internal/test/framework/windowsvm.go

```diff
@@ -38,70 +44,128 @@ type TestWindowsVM interface {
 	// RetrieveDirectories recursively copies the files and directories from the directory in the remote Windows VM
 	// to the given directory on the local host.
 	RetrieveDirectories(string, string) error
-	// Destroy destroys the Windows VM
-	// TODO: Remove this and move it to framework or other higher level object capable of doing deletions.
-	//		jira: https://issues.redhat.com/browse/WINC-243
-	Destroy() error
 	// BuildWMCB returns the value of buildWMCB. It can be used by WSU to decide if it should build WMCB before using it
 	BuildWMCB() bool
 	// SetBuildWMCB sets the value of buildWMCB. Setting buildWMCB to true would indicate WSU will build WMCB instead of
 	// downloading the latest as per the cluster version. False by default
 	SetBuildWMCB(bool)
 	// Compose the Windows VM we have from WNI
-	types.WindowsVM
+	windows.WindowsVM
```

Comment:
This is a bit confusing now. You have both  the struct and the interface in here. Why is that?